configs/recognition/c3d/c3d_sports1m_gpu_normalize_16x1x1_45e_ucf101_rgb.py

```diff
@@ -0,0 +1,104 @@
+_base_ = '../../_base_/models/c3d_sports1m_pretrained.py'
+
+module_hooks = [
+    dict(
+        type='GPUNormalize',
+        hook_pos='forward_pre',
+        input_format='NCTHW',
+        mean=(104, 117, 128),
+        std=(1, 1, 1))
```

Comment:
Where do the values of mean and std come from?

Reply:
This is the average value of each channel of all ucf101 images used by the author in the original paper. We also used this parameter to train the model before.

Reply:
ok, the std is not correct obviously